scripts/get_grants.py

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+import pandas as pd
+
+import os
+
+from datascience.warehouse.warehouse import FortyTwo
```

Comment:
datascience is not installed though?

Reply:
Yep, and it won't be because it's private. Maybe I'll add a `requirements-private` or something? What do you think?

Reply:
or a make command that installs it for those that have access. it can be separate from `make virtualenv`. also a `try except` in the script to remind us to install it :)

Reply:
Done https://github.com/wellcometrust/grants_tagger/pull/158/commits/ec4259e68434286256b624e5b6f3db8570bc3dd2